Misc/NEWS.d/next/Build/2022-10-28-22-24-26.gh-issue-98831.IXRCRX.rst

```diff
@@ -1 +1,10 @@
-We have new tooling, in ``Tools/cases_generator``, to generate the interpreter switch from a list of opcode definitions.
+We have new tooling, in ``Tools/cases_generator``,
```

Comment:
```suggestion
Add new tooling, in ``Tools/cases_generator``,
```

Conventional framing for a NEWS entry, per the devguide, is imperative (or at least some form of "Add(ed|s)/Removes/Changes etc).